config/locales/en.yml

```diff
@@ -1375,6 +1423,12 @@ en:
       policies:
         update: "Update Policy Chain"
     labels:
+      account:
+        payment_gateway_type: Gateway
+
+      billing_strategy:
+        numbering_period: Billing periods for invoice ids
```

Comment:
This translation doesn't seem to be kicking in?...
![image](https://github.com/3scale/porta/assets/1270328/763d27b8-4a3e-45af-ad1d-989bafdef8fd)


Also, we could:
- say IDs instead
- maybe add a hint for this field to help users understand what it means (an example of generated IDs)

Reply:
In addition to the hint: _It will only change the friendly ID in invoices, not the billing period._

What do you thing about using the following labels?
<img width="480" alt="Screenshot 2023-10-23 at 13 55 03" src="https://github.com/3scale/porta/assets/11672286/c9d59611-2173-4b3e-b13b-269a7645eff7">

Reply:
Ideally I would add a popover with some info taken from [the docs](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.13/html/admin_portal_guide/configure-billing#billing_periods_for_invoice_ids_charging_gateway).
 
<img width="502" alt="Screenshot 2023-10-23 at 13 47 50" src="https://github.com/3scale/porta/assets/11672286/ba353e34-ec66-4dbf-bdc8-7f03ebd351c1">